jekyll/_cci2/using-orbs.md

```diff
@@ -0,0 +1,158 @@
+
+<aside class="notice">
+You must invoke jobs in the workflow stanza of `config.yml`, making sure to pass any necessary parameters as subkeys to the job.
+</aside>
+
+### Executors
+{:.no_toc}
+
+Executors define the environment in which the steps of a job will be run. When you declare a `job` in CircleCI configuration, you define the type of environment (e.g. `docker`, `machine`, `macos`, etc.) to run in, in addition to any other parameters of that environment, such as:
+
+* environment variables to populate
+* which shell to use
+* what size `resource_class` to use
+
+When you declare an executor in a configuration outside of `jobs`, you can use these declarations for all jobs in the scope of that declaration, enabling you to reuse a single executor definition across multiple jobs.
+
+Each executor definition includes the subset of the children keys of a `job` declaration related to setting the environment for a job to execute. This means it does not include `steps`. The subset can be one or more of the following keys:
+
+* `docker`, `machine`, or `macos`
+* `environment`
+* `working_directory`
+* `shell`
+* `resource_class`
+
+The example below shows a simple example of using an executor:
+
+```
+version: 2.1
+executors:
+  my-executor:
+    docker:
+      - image: circleci/ruby:2.4.0
+
+jobs:
+  my-job:
+    executor: my-executor
+    steps:
+      - run: echo outside the executor
+ ```
+
+Notice in the above example that the executor `my-executor` is passed as the single value of the key `executor`. Alternatively, you can pass `my-executor` as the value of a `name` key under `executor`. This method is primarily employed when passing parameters to executor invocations. An example of this method is shown in the example below.
+
+```
+jobs:
+  my-job:
+    executor:
+      name: my-executor
+    steps:
+      -run: echo outside the executor
+```
+
+
+
+## Key Concepts
+
+Before using Orbs, you should first familiarize yourself with some basic core concepts of Orbs and how they are structured and operate. Gaining a basic understanding of these core concepts will enable you to leverage Orbs and use them easily in your own environments.
+
+### Development vs. Production Orbs
+{:.no_toc}
+
+Orbs can be published either as ```dev:foo``` or as a semantically versioned production orb. Development Orbs are mutable and expire, whereas Production Orbs are immutable and durable.
+
+### Certified vs. 3rd Party Orbs
+{:.no_toc}
+
+CircleCI will make available a number of individual Orbs that have been tested and certified to work with the platform. These orbs will be treated as part of the platform; all other orbs are considered 3rd party Orbs.
+
+<aside class="notice">
+All Orbs are open, meaning that anyone can use them and see their source. Also, only the ```circleci``` namespace is certified.
```

Comment:
I don't think we should codify "only the `circleci` namespace" here -- that will change at some point, and any investigation of orb lists will already enforce the distinction, so this isn't a durable or necessary caveat.

Reply:
removed